src/deploy/NVA_build/noobaa_deploy_k8s.sh

```diff
@@ -50,6 +54,13 @@ function usage(){
     echo "--num-pvs         -   number of PVs to provisons as backend storage. Minimum value is 3. default is 3"
     echo "--class           -   storageClass to use for PVs"
     # echo "--pv              -   when deploying a new system also provision kubernetes PVs to use as backend storage for objects. default is false"
+    echo "cloud-add arguments:"
```

Comment:
flags are a bit confusing if not using AWS

Reply:
I agree, but I thought it still made more sense than calling it `--cloud-identity` like in the internal API.
Regarding sigver I can rename to `--cloud-aws-sigver` if you think its more clear.